bin/run

```diff
@@ -3,7 +3,27 @@
 const oclif = require('@oclif/core')
 const { checkVersion } = require('../lib/request.js')
 
+const CODE_TO_ERROR_MESSAGE = {
+  Forbidden: 'You do not have permission to access this resource.',
+  InvalidAuth: 'Your credentials may have expired. Please try logging in again.',
+  EmailNotVerified: 'The email for this account has not been verified. Please log in to https://app.jamsocket.com to finish setting up your account.',
+  NoAccountFoundForUser: 'No account found for this user. Please log in to https://app.jamsocket.com to finish setting up your account.',
+  UsageLimit: 'You have exceeded the usage limit for this account. See https://https://docs.jamsocket.com for more information on account limits.',
+  UserNotEnabled: 'This user account has been disabled. Please contact support.',
```

Comment:
Support email here?

Reply:
I think I'll just change this to "Please contact us for more information."